File: DataTierGeneratorPlusTester/Cs_BuiltIn/Test_masterServiceClientExtensions.cs
using System.Collections.Generic;
using Cs_BuiltIn.Test_masterService;

namespace Cs_BuiltIn 
{

	internal static class Test_masterServiceClientExtensions
	{

		/// <summary>
		/// Converts from Array of Test_masterContract to List of Test_masterContract.
		/// </summary>
		internal static BindingListView<Test_masterContract> ToBindingListViewOfContract(this Test_masterContract[] infoList) 
		{ 
			BindingListView<Test_masterContract> returnValue = new BindingListView<Test_masterContract>();
			foreach (Test_masterContract info in infoList) 
			{
				returnValue.Add
					(
					info
					);
			}
			return returnValue;
		}
	}
}

File: DataTierGeneratorPlusTester/VB_BuiltIn/Test_masterServiceClientExtensions.vb
Imports System.Runtime.CompilerServices
Imports System.Collections.Generic
Imports VB_BuiltIn.Test_masterService

'Namespace VB_BuiltIn 

	Friend Module Test_masterServiceClientExtensions 

		''' <summary>
		''' Converts from Array of Test_masterContract to List of Test_masterContract.
		''' </summary>
		<Extension()> _
		Friend Function ToBindingListViewOfContract(ByVal infoList As Test_masterContract()) As BindingListView(Of Test_masterContract)  
			Dim returnValue As New BindingListView(Of Test_masterContract) 
			For Each info As Test_masterContract In infoList
				returnValue.Add _
					( _
					info _
					)
			Next
			Return returnValue
		End Function
	End Module
'End Namespace
